core/src/main/java/com/linecorp/armeria/common/HttpStatus.java

```diff
@@ -496,35 +496,35 @@ public boolean isContentAlwaysEmpty() {
      * Returns whether the {@link HttpStatus} is an information, with a status code of 1XX.
      */
     public boolean isInformational() {
-        return codeClass().equals(HttpStatusClass.INFORMATIONAL);
+        return codeClass == HttpStatusClass.INFORMATIONAL;
```

Comment:
Realized that we already pre-determined the class of all status instances, so I optimized it like this.